content/recipes/swagger.md

```diff
@@ -200,71 +300,33 @@ Now you can start your server with the following command:
 $ npm run start
 ```
 
-Navigate to `http://localhost:3000/api/cats` to see SwaggerUI for your cats:
+Navigate to `http://localhost:3000/api/cats` to see Swagger UI for your cats:
 
 <figure><img src="/assets/swagger-cats.png" /></figure>
 
-While `http://localhost:3000/api/dogs` will expose a SwaggerUI for your dogs:
+While `http://localhost:3000/api/dogs` will expose a Swagger UI for your dogs:
```

Comment:
While `http://localhost:3000/api/dogs` will expose a Swagger UI for your dogs:

->

In turn, `http://localhost:3000/api/dogs` will expose the Swagger UI for dogs: